Userland/Applications/SoundPlayer/PlaybackManager.cpp

```diff
@@ -6,23 +6,17 @@
  */
 
 #include "PlaybackManager.h"
+#include <LibThreading/Thread.h>
```

Comment:
Is this left over from a change that didn't make it into this PR?

Reply:
Probably helpful clangd, yeah (I don't recall ever trying explicit multi-threading in SP...)